Update learn-about-office-365-germany.md (#926)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/admin-overview/learn-about-office-365-germany.md

description: "Learn about Office 365 Germany."
---

> >[!Note]

Reviewer: ```suggestion
>[!NOTE]
```

> >Office 365 Germany is no longer accepting new customers or deploying new services. The new cloud regions in Germany, available in the first quarter of 2020 for Microsoft 365 and Office 365, will provide both core customer data residency within Germany, as well as full connectivity to Microsoft's global cloud network. 

Reviewer: ```suggestion
> Office 365 Germany is no longer accepting new customers or deploying new services. The new cloud regions in Germany, available in the first quarter of 2020 for Microsoft 365 and Office 365, will provide both core customer data residency within Germany, as well as full connectivity to the Microsoft global cloud network. 
```

> >

Reviewer 2: Per MS style guidelines, we can't use the possessive with "Microsoft" or product names. Please change "Microsoft's global cloud network" to "the Microsoft global cloud network."

# Learn about Office 365 Germany

> Office 365 Germany was a differentiated option to the Office 365 services available across Europe. It was supposed to help address the needs of the most regulated customers in Germany, the European Union (EU), and the European Free Trade Association (EFTA) by delivering our industry-leading productivity services for digital work, from German datacenters, with data residency in Germany, and strict data access and control measures via a unique data trustee model governed under German law.

Reviewer 2: Please change "It was supposed to help" to "It helped".

> Office 365 Germany was a differentiated option to the Office 365 services available across Europe. It was supposed to help address the needs of the most regulated customers in Germany, the European Union (EU), and the European Free Trade Association (EFTA) by delivering our industry-leading productivity services for digital work, from German datacenters, with data residency in Germany, and strict data access and control measures via a unique data trustee model governed under German law.

Reviewer: ```suggestion
Office 365 Germany was a differentiated option to the Office 365 services available across Europe. It helped to help address the needs of the most regulated customers in Germany, the European Union (EU), and the European Free Trade Association (EFTA) by delivering our industry-leading productivity services for digital work, from German datacenters, with data residency in Germany, and strict data access and control measures via a unique data trustee model governed under German law.
```

  
The data trustee, T-Systems International, an independent German company and subsidiary of Deutsche Telekom, controls physical and logical access to customer data. Customer data cannot be accessed without approval from or supervision by the data trustee, which is governed by German law.
  

> But customers need have shifted, and the isolation of Office 365 Germany imposed limits on its ability to address the flexibility and consistency that customers desire. Since August 2018 we are no longer accpeting new customers or deploying any new services from the currently available Microsoft Cloud Germany. For further details see this blog post: [Microsoft to deliver cloud services from new datacentres in Germany in 2019 to meet evolving customer needs](https://go.microsoft.com/fwlink/p/?linkid=839016).

Reviewer 2: Please change "But customers need have shifted," to "Customer needs have shifted,".

> But customers need have shifted, and the isolation of Office 365 Germany imposed limits on its ability to address the flexibility and consistency that customers desire. Since August 2018 we are no longer accpeting new customers or deploying any new services from the currently available Microsoft Cloud Germany. For further details see this blog post: [Microsoft to deliver cloud services from new datacentres in Germany in 2019 to meet evolving customer needs](https://go.microsoft.com/fwlink/p/?linkid=839016).

Reviewer: ```suggestion
Customers needs have shifted, and the isolation of Office 365 Germany imposed limits on its ability to address the flexibility and consistency that customers desire. Since August 2018, we are no longer accepting new customers or deploying any new services from the currently available Microsoft Cloud Germany. For further details, see this blog post: [Microsoft to deliver cloud services from new datacentres in Germany in 2019 to meet evolving customer needs](https://go.microsoft.com/fwlink/p/?linkid=839016).
```
